contracts/ZeroCollateral.sol

```diff
@@ -131,12 +131,64 @@ contract ZeroCollateralMain is ERC20Detailed, ERC20, MinterRole, Chainlink {
         decimals
     )
     {
+        require((daiAddress != address(0)) && (daoAddress != address(0)), "ZeroCollateral::constructor - invalid addresses");
```

Comment:
This revert message is over 32 characters long and so has to be handled in a special way. This means that the gas costs are more expensive. It would be better to try and have a message that was 32 characters or less.

Reply:
Same with the other revert messages

Reply:
Right. To avoid potential similar issues, we may define a naming convention / standard errors codes or similar.